subsys/bluetooth/host/settings.c

```diff
@@ -233,7 +241,13 @@ static int commit(void)
 {
 	int err;
 
-	BT_DBG("");
+	if (!atomic_test_and_set_bit(bt_dev.flags, BT_DEV_ENABLE)) {
```

Comment:
Same here. Or is there something I'm missing, why you want to set the ENABLE bit here?